src/components/Books.js

```diff
@@ -21,18 +21,8 @@ const Books = () => {
       //   console.log(error)
       // })
     },[])
-
-    const handleSignOut = async () => {
-        try{
-          await Auth.signOut()
-          history.push("/signin")
-        }
-        catch(error) {
-          console.log("Error in Logging Out ",error);
-        }
-      }
     
-    const handleSubmit = () => {
+  const handleSubmit = () => {
     axios.get(`https://www.googleapis.com/books/v1/volumes?q=${searchKey}&key=${process.env.REACT_APP_API_KEY}`) 
```

Comment:
You can start hooks based api library. 
check this out https://www.npmjs.com/package/swr